tests/integration_tests/dashboard_tests.py

```diff
@@ -58,39 +58,36 @@
 class TestDashboard(SupersetTestCase):
     @pytest.fixture
     def load_dashboard(self):
-        with app.app_context():
```

Comment:
The app context is not needed within the confines of a derived Flask-Testing class.